app/src/main/java/com/nmakademija/nmaakademija/fragment/UsersFragment.java

```diff
@@ -86,8 +86,9 @@ public int compare(Section section, Section t1) {
                                 }
                             });
                             String[] sectionsString = new String[sections.size() + 1];
+                            int i = 1;
                             for (Section section : sections) {
-                                sectionsString[section.getId()] = section.getName();
+                                sectionsString[i++] = section.getName();
```

Comment:
Avoid such mutation. `ArrayAdapter` has contructor with `List`. So this can be rewritten:
```java
                            List<String> sectionNames = new ArrayList<>();
                            sectionNames.add(getResources().getString(R.string.all_academics));

                            for (Section section : sections) {
                                sectionNames.add(section.getName());
                            }

                            ArrayAdapter<String> spinnerArrayAdapter = new ArrayAdapter<>(getContext(), 
                                    android.R.layout.simple_spinner_item, sectionNames);
```